zcash_client_backend/src/fees.rs

```diff
@@ -301,6 +348,51 @@ impl Default for DustOutputPolicy {
     }
 }
 
+#[cfg(feature = "transparent-inputs")]
+#[derive(Clone, Debug, Default, PartialEq, Eq)]
+pub struct EphemeralParameters {
+    ignore_change_memo: bool,
+    ephemeral_input_amount: Option<NonNegativeAmount>,
+    ephemeral_output_amount: Option<NonNegativeAmount>,
+}
+
+#[cfg(feature = "transparent-inputs")]
+/// `EphemeralParameters` can be used to specify variations on how balance
+/// and fees are computed that are relevant to transactions using ephemeral
+/// transparent outputs.
```

Comment:
Representing this as a struct makes it easier to change without disrupting all call sites.